include/openenclave/bits/result.h

```diff
@@ -265,10 +265,20 @@ typedef enum _oe_result
     OE_UNSUPPORTED_ENCLAVE_IMAGE,
 
     /**
-     * The CRL for a PCK certification expired
+     * The CRL for a PCK certificate has expired.
```

Comment:
I think "PCK" should be deleted from line 268.  The name in 270 doesn't have it, and PCK is very SGX specific unlike the name in line 270.

Reply:
Removed